### PR TITLE
patch project identity update

### PR DESCRIPTION
**Pull request by maidul98**

### CODE REVIEW

File: backend/.eslintrc.js [hunk 1]
```diff
@@ -38,6 +38,7 @@ module.exports = {
     "@typescript-eslint/no-empty-function": "off",
     "@typescript-eslint/no-unsafe-enum-comparison": "off",
     "no-void": "off",
+    "no-await-in-loop": "off",
```
Comment (DanielHougaard) [inline backend/.eslintrc.js:41]: I wouldn't turn off this rule. Instead I would write for loops like this:

```js
for await (const item of items) {
   await something(item);
}
```

This won't give lint errors

File: backend/src/services/identity-project/identity-project-service.ts [hunk 2]
```diff
@@ -135,16 +136,18 @@ export const identityProjectServiceFactory = ({
...
-      ActorType.IDENTITY,
-      projectIdentity.identityId,
-      projectIdentity.projectId,
-      actorAuthMethod,
-      actorOrgId
-    );
-    const hasRequiredPriviledges = isAtLeastAsPrivileged(permission, identityRolePermission);
-    if (!hasRequiredPriviledges)
-      throw new ForbiddenRequestError({ message: "Failed to delete more privileged identity" });
+    for (const { role: requestedRoleChange } of roles) {
```
Comment (DanielHougaard) [inline backend/src/services/identity-project/identity-project-service.ts:139]: ```for await (const { role: requestedRoleChange } of roles)```

**Review by DanielHougaard — CHANGES_REQUESTED** [review]
Looks good, requested a small change

File: backend/.eslintrc.js (hunk 1 above)
Comment (maidul98) [reply backend/.eslintrc.js:41]: sweet, let me change it

**Review by DanielHougaard — APPROVED** [review]

**Pull request merged**